test_scripts/Defects/4_5/1873_UnsubscribeVD_OnVD_Parameters_empty_in_policy_table.lua

```diff
@@ -0,0 +1,102 @@
+---------------------------------------------------------------------------------------------
+-- GitHub issue: https://github.com/SmartDeviceLink/sdl_core/issues/1873
+-- In case
+-- mobile app sends SendLocation_request to SDL
+-- and this request is allowed by Policies for this mobile app
+-- and "parameters" field is empty at PolicyTable for SendLocation RPC
+-- SDL must:
+-- respond with "DISALLOWED, success:false", "info: Requested parameters are disallowed by Policies"
+-- SDL must NOT:
+-- transfer this request to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonDefects = require('test_scripts/Defects/4_5/commonDefects')
+local json = require("json")
+
+--[[ Local Variables ]]
+local gpsDataResponse = {
+  longitudeDegrees = 100,
+  latitudeDegrees = 20,
+  utcYear = 2050,
+  utcMonth = 10,
+  utcDay = 30,
+  utcHours = 20,
+  utcMinutes = 50,
+  utcSeconds = 50,
+  compassDirection = "NORTH",
+  pdop = 5,
+  hdop = 5,
+  vdop = 5,
+  actual = false,
+  satellites = 30,
+  dimension = "2D",
+  altitude = 9500,
+  heading = 350,
+  speed = 450
+}
+
+--[[ Local Functions ]]
+local function ptuUpdateFuncDissalowedRPC(tbl)
+  local VDgroup = {
+    rpcs = {
+      SubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = {"gps"}
+      },
+      OnVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = json.EMPTY_ARRAY
+      },
+      UnsubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = json.EMPTY_ARRAY
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+  if tbl.policy_table.functional_groupings["SendLocation"] then
+    tbl.policy_table.functional_groupings["SendLocation"] = nil
+  end
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups =
+  {"Base-4", "NewTestCaseGroup"}
+end
+
+local function SubscribeVD(self)
+  local cid = self.mobileSession1:SendRPC("SubscribeVehicleData", {gps = true})
+  EXPECT_HMICALL("VehicleInfo.SubscribeVehicleData")
+  :Do(function(_,data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+local function UnsubscribeVD(self)
+  local cid = self.mobileSession1:SendRPC("UnsubscribeVehicleData", {gps = true})
+  EXPECT_HMICALL("VehicleInfo.UnsubscribeVehicleData")
+  :Times(0)
+  commonDefects.delayedExp()
```

Comment:
@GetmanetsIrina I would propose set delay timeout explicitly for improve readability of script.